src/components/avatar/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface AvatarProps extends React.ComponentPropsWithoutRef<'div'> {
+  firstName: string;
+  lastName: string;
+  size?: 'sm' | 'md' | 'lg';
```

Comment:
Can we make an enum for sizes?

Reply:
And can we pass width & height props as optional props for different size avatar?

Reply:
@dhruvilsaleshandy it is the same as an enum only, which means the `size` prop can have any of those values, the issue with an enum is whenever we want to pass the value we first have to import it. This is a much better approach.

<img width="125" alt="image" src="https://user-images.githubusercontent.com/105193391/209768267-5e6a2de7-ad31-4129-9aa6-260aec8efe38.png">

Reply:
> And can we pass width & height props as optional props for different size avatar?

I had a chat with the design team, the size will be from one of the defined sizes, so there is no need for custom width and height. Although If any such case happened in the future then it will be suggested that we should negotiate the design and use the pre-defined sizes for consistency